File: app/src/main/java/com/github/kazy1991/remee/application/RemeeApplication.java
package com.github.kazy1991.remee.application;

import android.app.Application;

import com.facebook.drawee.backends.pipeline.Fresco;
import com.github.kazy1991.dependencykit.DependencyKit;
import com.github.kazy1991.remee.module.RemeeModule;

public class RemeeApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        DependencyKit.configure(new RemeeModule(this));
        Fresco.initialize(this);
    }
}

File: app/src/main/java/com/github/kazy1991/remee/application/RemeeApplication.kt
package com.github.kazy1991.remee.application

import android.app.Application

import com.github.kazy1991.dependencykit.DependencyKit
import com.github.kazy1991.remee.module.RemeeModule

class RemeeApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        DependencyKit.configure(RemeeModule(this))
    }
}
